cmd/jujud/agent/machine/manifolds.go

```diff
@@ -488,6 +497,17 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 			NewWorker:      globalclockupdater.NewWorker,
 			UpdateInterval: globalClockUpdaterUpdateInterval,
 			BackoffDelay:   globalClockUpdaterBackoffDelay,
+			Logger:         loggo.GetLogger("juju.worker.globalclockupdater.mongo"),
+		}),
+		// We also run another clock updater to feed time updates into
+		// the lease FSM.
+		leaseClockUpdaterName: globalclockupdater.Manifold(globalclockupdater.ManifoldConfig{
```

Comment:
I started doing that initially but then moved to doing this instead. My thinking was: we want the updates to each clock to be independent (can't have one of them being slow preventing updates to the other one), so that means that they would need to run as separate goroutines inside the worker each talking to their own updater. At that point it seemed cleaner just to have two instances of the worker, each talking to a specific updater.